components/plugin/index.js

```diff
@@ -1,14 +1,19 @@
 import Highlighter from 'react-highlighter'
 import styles from './plugin.module.css'
+import Image from 'next/image'
 
 const MAX_COLORS = 4
 
 export default ({ featured, name, description, colors, preview, query }) =>
   featured ? (
-    <div
-      className={styles.featured}
-      style={{ backgroundImage: `url(${preview})` }}
-    >
+    <div className={styles.featured}>
+      <Image
+        src={preview.src}
+        layout="fill"
+        objectFit="cover"
+        objectPosition="center top"
+        unoptimized
```

Comment:
Seems the gifs on plugin pages aren't unoptimized, i.e.: https://hyper-site-git-main.vercel.vercel.app/store/hyper-search